pymeasure/instruments/srs/sr830.py

```diff
@@ -583,3 +588,39 @@ def snap(self, val1="X", val2="Y", *vals):
 
         command = "SNAP? " + ",".join(vals_idx)
         return self.values(command)
+
+    def save_setup(self, setup_number: int):
+        """Save the current instrument configuration (all parameters) in a memory
+        referred to by an integer
+
+        The integer must be comprised between 1 and 9 (included)
+
+        :param setup_number: the integer referring to the memory (between 1 and 9 (included))
+        """
+        if 1 <= setup_number <= 9:
+            self.write(f'SSET{setup_number:d};')
+
+    def load_setup(self, setup_number: int):
+        """ Load a previously saved instrument configuration from the memory referred
+        to by an integer
+
+        The integer must be comprised between 1 and 9 (included)
+
+        :param setup_number: the integer referring to the memory (between 1 and 9 (included))
+        """
+        if 1 <= setup_number <= 9:
+            self.write(f'RSET{setup_number:d};')
+
+    def start_scan(self):
+        """
+
+        :return:
+        """
```

Comment:
If you do not return anything, just remove that line. On the other hand, it is good to write something into the docstring, if you have one.

Also below.